Mathlib/Topology/ContinuousMap/Algebra.lean

```diff
@@ -617,6 +617,13 @@ def coeFnLinearMap : C(α, M) →ₗ[R] α → M :=
   { (coeFnAddMonoidHom : C(α, M) →+ _) with
     map_smul' := coe_smul }
 
+/-- Evaluation at a point, as a continuous linear map. -/
+@[simps apply]
+def evalCLM (x : α) : C(α, M) →L[R] M where
+  toFun f := f x
+  map_add' _ _ := add_apply _ _ x
+  map_smul' _ _ := smul_apply _ _ x
```

Comment:
It hardly matters, but you can get here in even fewer lines by composing `coeFnLinearMap` with `ContinuousLinearMap.proj`

Reply:
This approach doesn't quite give the one-liner, since `coeFnLinearMap` is a "plain" linear map without bundled continuity, so one has to put the continuity argument in manually. I'm also not quite sure that `@[simps]` would generate the correct `apply` lemma that way. Hence I'd rather leave it as-is for now.